modules/ROOT/pages/apicat-create-descriptor-file-manually.adoc

```diff
@@ -0,0 +1,226 @@
+
+You can set version strategy conditions for specific APIs or for all APIs described in the descriptor file (global). If both types of conditions are used, the specific override the global. 
+
+Following is an example `versionStrategyConditions` section:
+
+----
+    versionStrategyConditions:
+      minorIncrease:
+        branches:
+          - main
+        tags:
+          - support
+      snapshot:
+        branches:
+          - develop
+----
+
+== contact
+
+The `contact` section of the schema includes the name and email ID for main contact for the autocataloging. If none are included, these default to `unknown`.   
+
+== projects 
+
+The `projects` section of the schema lists the API projects to be cataloged along with the details needed to catalog them. 
+
+spec:: The location of the specification file that describes the API. 
+
+assetId:: Specifies the ID of the asset to be published in Exchange. 
+
+documentation:: A list of key and value pairs that specify the name of the documentation page and file location for the API.
+These pairs define the order of the documentation pages in Anypoint Platform.
++
+Format: <Name of the documentation page>:<File Location>
++
+If this option is not specified, no documentation pages are imported. 
+
+categories:: A list of key and value pairs that specify the category name and value. These pairs are validated against the categories that exist in the Anypoint Platform organization.
++
+Format: <Category Name>: <Value(s)>
++
+If this option is not specified, no categories are added.
+
+customFields:: A list of key and value pairs that specify the field and value. These pairs are validated against the custom fields and field types that exist in the Anypoint Platform organization.
++
+Format: <Field>: <Value(s)> 
++
+If this option is not specified, no custom fields are added.
+
+tags:: A list of free text strings. 
++
+If this option is not specified, no tags are added.
++
+version:: The version of the API.
++
+If this is not specified, the version is set by the version strategy.
++
+versionStrategy::
++
+You can set the versioning strategy for the assets at the global or API project level. If a version strategy is not specified, the patch version is incremented by one. 
++
+Following are the possible values:
++
+* *majorIncrease:* Searches for the latest version that matches the `version` field in the descriptor and increases the major version. If the asset is in the development lifecycle state, the version is increased and the asset stays in development. If the asset is a stable version, a new stable version is published.
```

Comment:
I feel like it might be overboard at this point, but instead of "a new stable version is published" can it be "Exchange (or whatever) publishes a new stable version"

this would be throughout this section. you decide